### PR TITLE
Added the ability to supply multiple hosts for the same domain

### PR DESCRIPTION
**Pull request by bspradling**
This change allows for the user to supply multiple hosts (or subdomains) in their auth.json for them all to get updated to the public ip.

### CODE REVIEW

**Comment by cinupina** [comment]
👍

**Comment by cinupina** [comment]
Can you update the example config in readme too?

File: src/index.js [hunk 1]
```diff
@@ -27,9 +28,18 @@ function getPublicIp(callback) {
...
-function getDnsRecords(domain, host, callback) {
-  const path = `/v1/domains/${domain}/records/A/${host}`;
-  requestApi('GET', path, null, true, callback);
+function getDnsRecords(domain, hosts, callback) {
+  async.map(hosts, function (host, callback) {
+    const path = `/v1/domains/${domain}/records/A/${host}`;
+    requestApi('GET', path, null, true, (err, response) => {
+      if (err){
+        return;
+      }
```
Comment (cinupina) [inline src/index.js:37]: Don't you want ' return void callback(err);'?

Comment (bspradling) [reply src/index.js:37]: Yes, though wouldn't it be return callback(err);?

Comment (cinupina) [reply src/index.js:37]: That works too. Return void just makes sure it's not actually returning anything.

**Comment by cinupina** [comment]
Looks great!

**Pull request merged**